hie-ghc-mod/Haskell/Ide/GhcModPlugin.hs

```diff
@@ -44,10 +44,10 @@ ghcmodDescriptor = PluginDescriptor
       :& buildCommand infoCmd (Proxy :: Proxy "info") "Look up an identifier in the context of FILE (like ghci's `:info')"
                      [".hs",".lhs"] (SCtxFile :& RNil)
                      (  SParamDesc (Proxy :: Proxy "expr") (Proxy :: Proxy "The EXPR to provide info on") SPtText SRequired
-                     :& RNil)
+                     :& RNil) Safe
 
       :& buildCommand typeCmd (Proxy :: Proxy "type") "Get the type of the expression under (LINE,COL)"
-                     [".hs",".lhs"] (SCtxPoint :& RNil) RNil
+                     [".hs",".lhs"] (SCtxPoint :& RNil) RNil ChangeCurrent
```

Comment:
I don't think this actually changes anything, but if you add some code, it may need to be saved before receiving proper line and column numbers.